docs/products/postgresql/concepts/pg-shared-buffers.rst

```diff
@@ -0,0 +1,166 @@
+PostgreSQL® shared buffers
+===========================
+
+In a very simplified view of PostgreSQL® memory, when a query is executed, there are 2 primary memory allocations that will drastically impact query performance: ``shared_buffers`` and ``work_mem``.
+
+To paraphrase the PostgreSQL® documentation_:
+  The ``shared_buffers`` parameter controls the amount of memory allocated to the database server to use for disk page caching.
+  To get good performance, on a dedicated database server has 1 gigabyte or more of RAM, it is usually necessary to set this value to `~25%` of the system memory. 
+  For systems with less than 1gb of RAM, a smaller portion of RAM is preferable to allow for the operating system.
+
+It's important to note that ``shared_buffers`` is not allocated per-session or per-user and it is allocated only once, at startup.
+
+**It is shared among all sessions, and is useable for every worker and ,therefore, every query.**
```

Comment:
It is shared among all sessions and useable for each worker and, therefore, each query.